.github/CODEOWNERS

```diff
@@ -1,29 +1,22 @@
 # File extension owners
 
-*.sol @yorhodes @tkporter @aroralanuk @nbayindirli
-*.ts @yorhodes @jmrossy @nbayindirli
-*.rs @tkporter @daniel-savu
-*.md @Skunkchain @avious00
+*.sol @yorhodes @aroralanuk @ltyu
+*.ts @yorhodes @jmrossy
+*.rs @tkporter @daniel-savu @ameten
 
 # Package owners
```

Comment:
do you reckon it's worth adding owners for `.github/actions` and `.github/workflows` as well?